src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -35,4 +35,14 @@ public void runTurret(double speed) {
 
     turretMotor.set(speed);
   }
+
+  public String turretTestMethod(){
+
+    return "Hello, test!";
```

Comment:
Daha iyi bir test yazabilir miyiz? Motor hızı falan kontrol edecek en azından? :D

Reply:
PWM motor olmadığı için robotta yazmadım ama istersen rastgele bir tane pwm sürücü tanımlayıp yaparım abi

Reply:
hallettim

Reply:
Ek sürücü tanımlamayalım ya, çünkü bu robotun içinde çalışacak kod, boşta sürücü olmasını istemeyiz. Kusura bakma biraz yanış yönlendirmiş oldum. CAN motorlara test yazamıyor muyuz? Eğer yazamıyorsak şu an için, boş string yerine en azından reset veya sıfırlama fonksiyonlarını falan test edebiliriz (örn. drivesubsystem odometri hesaplama)

Reply:
@vatanaksoytezer CAN simülasyonları wpilibde mevcut değil ctre için ayrıca bakacağım ona, asıl testi yazarken entegre ederiz diye düşünmüştüm.